examples/tensorflow/object_detection/preprocessing/yolo_v4_preprocessing.py

```diff
@@ -1,358 +1,362 @@
-        groundtruths = dataloader_utils.pad_groundtruths_to_fixed_size(groundtruths, self._max_num_instances)
-        labels.update(groundtruths)
-
-        return image_data, labels
-
-    def _tfds_decoder(self, features_dict):
-
-        def _decode_image(features):
-            image = tf.image.decode_jpeg(features['image'], channels=3, dct_method='INTEGER_ACCURATE')
-            image.set_shape([None, None, 3])
-            return image
-
-        def _convert_labels_to_91_classes(features):
-            # 0..79 --> 0..90
-            match = tf.constant([1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                                 11, 13, 14, 15, 16, 17, 18, 19, 20, 21,
-                                 22, 23, 24, 25, 27, 28, 31, 32, 33, 34,
-                                 35, 36, 37, 38, 39, 40, 41, 42, 43, 44,
-                                 46, 47, 48, 49, 50, 51, 52, 53, 54, 55,
-                                 56, 57, 58, 59, 60, 61, 62, 63, 64, 65,
-                                 67, 70, 72, 73, 74, 75, 76, 77, 78, 79,
-                                 80, 81, 82, 84, 85, 86, 87, 88, 89, 90], dtype=tf.int64)
-
-            labels = features['objects']['label']
-            labels = tf.gather(match, labels, axis=None)
-            return labels
-
-        image = _decode_image(features_dict)
-        if self._is_training:
-            labels = features_dict['objects']['label']
-        else:
-            labels = _convert_labels_to_91_classes(features_dict)
-
-        decoded_tensors = {
-            'image': image,
-            'source_id': tf.cast(features_dict['image/id'], tf.int32),
-            'groundtruth_classes': labels,
-            'groundtruth_is_crowd': features_dict['objects']['is_crowd'],
-            'groundtruth_area': features_dict['objects']['area'],
-            'groundtruth_boxes': features_dict['objects']['bbox'],
-        }
-        return decoded_tensors
-
-    def _pipeline_fn(self, dataset, decoder_fn):
-        if self._is_training:
-            preprocess_input_fn = self._parse_fn
-            preprocess_pipeline = lambda record: preprocess_input_fn(decoder_fn(record))
-            dataset = dataset.map(preprocess_pipeline, num_parallel_calls=self._num_preprocess_workers)
-            dataset = dataset.batch(self._global_batch_size, drop_remainder=True)
-
-            # part of preprocessing which requires batches
-            preprocess_input_fn2 = self._parse_fn2
-            dataset = dataset.map(preprocess_input_fn2, num_parallel_calls=self._num_preprocess_workers)
-        else:
-            preprocess_input_fn = self._parse_predict_data
-            preprocess_pipeline = lambda record: preprocess_input_fn(decoder_fn(record))
-            dataset = dataset.map(preprocess_pipeline, num_parallel_calls=self._num_preprocess_workers)
-            dataset = dataset.batch(self._global_batch_size, drop_remainder=True)
-        return dataset
+"""
```

Comment:
I don't know why git can't handle this diff

Reply:
The reason is file on develop has CRLF format, and my git configured to commit only in LF format. I have found `examples/tensorflow/object_detection/preprocessing/yolo_v4_preprocessing.py` and `tests/tensorflow/data/ac_configs/yolo_v4.yml` has CRLF file format, so I move them to default LF format thus this diff is correct

Reply:
cc: @negvet